packages/integration-tests/tests/config.ts

```diff
@@ -24,7 +24,7 @@ export class Config {
 
   private rootUrlPathFromEnv({HOST, PORT}: typeof EnvConfig): string {
     const scheme = HOST === 'localhost' ? 'http' : 'https'
-    return `${scheme}://${HOST}${PORT ? `:${PORT}` : ''}`
+    return `${scheme}://${HOST}${HOST === 'localhost' ? `:${PORT}` : ''}`
```

Comment:
Since we're reading the `.env` file, `PORT` was set to the `.env` value, even when we wanted to test against a different `HOST`. For example `HOST=action-staging.parabol.co yarn test` was trying to hit `https://action-staging.parabol.co:3000`.

Reply:
-1 The right thing to do then would be to unset PORT, wouldn't it? So the check should be `PORT ? `:${PORT}` : ''`

Reply:
port is a funny one because it's only exposed on the ingress for localhost (see https://github.com/ParabolInc/parabol/blob/f221880dac24332874939b6c0a3f7328f4b76023/packages/server/appOrigin.ts#L4)

gonna merge this as is because today we're just testing on localhost. if there's a use case that's still broken we can fix it then